src/components/specific/files/files-manager/files-manager-breadcrumb/FilesManagerBreadcrumb.scss

```diff
@@ -2,11 +2,10 @@
   &__path {
     display: inline-flex;
     align-items: center;
-    gap: calc(var(--spacing-unit) / 2);
+    gap: 15px;
   }
 
   &__file {
-    margin-left: calc(var(--spacing-unit) / 2);
```

Comment:
I got rid of this property, because I handle it dynamically in FilesManagerBreadcrumb component

Reply:
Thats fine.